src/main/java/ua/com/juja/microservices/keepers/slackbot/model/SlackParsedCommand.java

```diff
@@ -25,7 +28,10 @@ public SlackParsedCommand(UserDTO fromUser, String text, List<UserDTO> usersInTe
         this.fromUser = fromUser;
         this.text = text;
         this.usersInText = usersInText;
-
+        slackNamePattern = getProperty(
+                "src/main/resources/application.properties",
```

Comment:
bad path after compile we haven't directory src. Please watch the video and fix your code
https://www.youtube.com/watch?v=kQ-475fKKZ0